### PR TITLE
haechi/2

### LINKED ISSUE #85: 2 - newCurve() cause DoS by Front-Running

### PR DESCRIPTION
**Pull request by chanharryhk**
# Description
fixes #85 
# Integrations Checklist

- [ ] Have any function signatures changed? If yes, outline below.
- [ ] Have any features changed or been added? If yes, outline below.
- [ ] Have any events changed or been added? If yes, outline below.
- [ ] Has all documentation been updated?

# Changelog

## Events

### CODE REVIEW

**Comment by kafitz** [comment]
Is this to fix Haechi issue #2 or #3?

File: src/Orchestrator.sol [hunk 1]
```diff
@@ -79,6 +79,40 @@ library Orchestrator {
...
+
+        // includeAsset(
+        //     curve,
+        //     _numeraire, // numeraire
+        //     _numeraireAssim, // numeraire assimilator
+        //     _reserve, // reserve
+        //     _reserveAssim, // reserve assimilator
+        //     _reserve, // reserve approve to
+        //     5e17
+        // );
```
Comment (kafitz) [inline src/Orchestrator.sol:113]:
```suggestion
```

Did you want to keep this for anything?

**Comment by chanharryhk** [comment]
Yea youre right changed to #85

**Review by kafitz — APPROVED** [review]
Nice tests, LGTM

**Review by tommy4241 — APPROVED** [review]
implementation looks good

**Pull request merged**